### PR TITLE
Add `*-dot` icons

### PR DESCRIPTION
**Pull request by danielbayley**
These correspond to `debug-step-*` _[codicons](https://microsoft.github.io/vscode-codicons/dist/codicon.html)_

### CODE REVIEW

File: icons/arrow-down-dot.json
Comment (karsa-mistmere) [inline icons/arrow-down-dot.json:1]: Feels a bit incomplete and the names will clash should we decide to add other variations.
I'd suggest to go for a naming scheme like `arrow-up-from-dot` vs. `arrow-down-to-dot`.

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

File: icons/arrow-down-dot.json
Comment (danielbayley) [reply icons/arrow-down-dot.json:1]: > Feels a bit incomplete

What’s missing?

Comment (karsa-mistmere) [reply icons/arrow-down-dot.json:1]: What I meant by incomplete is that I could easily see how someone out there at some point might also want any other variation of `arrow-(up|down)-(from|to)-dot`, so we should make sure the current names won't clash with adding these later on.

File: icons/dot.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "step",
```
Comment (karsa-mistmere) [inline icons/dot.json:4]:
```suggestion
    "interpunct",
    "interpoint",
    "middot",
    "step",
```

**Review by ericfennis — APPROVED** [review]

**Review by karsa-mistmere — APPROVED** [review]

**Pull request merged**